l10n_it_withholding_tax_account_accountant/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,3 @@
+Questo modulo sistema il calcolo dell'importo delle registraioni contabili in caso di
+ritenuta d'acconto su ambienti in cui è installato il modulo l10n_it_withholding_tax
+e account_accountant.
```

Comment:
@LorenzoC0 puoi descrivere un po' più precisamente il caso d'uso specificando che riguarda la riconciliazione bancaria?

Reply:
Tipo
"Questo modulo sistema il calcolo degli importi da riconciliare durante la riconciliazione bancaria, in caso di
ritenuta d'acconto, su ambienti in cui sono installati i moduli l10n_it_withholding_tax
e account_accountant"